src/main/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotations.java

```diff
@@ -213,10 +231,56 @@ private VariantContext applyFilters(final VariantContext variant, final Set<Stri
         return variantContextBuilder.make();
     }
 
+    private void buildArHetByGene(VariantContext variant) {
```

Comment:
Should this be in its own class?. Having AR specific code in the `FilterFuncotations` class seems like it breaks encapsulation.

Reply:
Yeah, it could be. There's not currently a 'good' place for it as this can't be a `FuncotationFilter`.

Reply:
Yeah - it may be worth designing another class hierarchy for filters that require two passes.  That way the next time someone has to make a filter that requires work on the first pass it'll be easier.

Maybe as a subclass of `FuncotationFilter`?